decidim/decidim
# Redesign: remove show page from Process in Admin

#### :tophat: What? Why?

We have a show page from Processes but we're not happy with the current approach, as there are some missing elements from the Figma mockup. 

On @decidim/product we've talked that until this page has at least some other features, currently is not usable. What we missed was the "Last activity" block from the mockup, and also having the links to publish/unpublish/configure/manage Components. 

On the other hand, with the introduction of the sidebar the links that it has are redundant, so we need to iterate also in the sidebar menu and the relation with this page (i.e. having a collapsed Components section there?) 

Finally, another problem with this particular page is that its only implemented in Processes, and it makes weird the navigation of the other spaces. The first time I got into the admin and started clicking things, I could not know what was happening with Assemblies, and it was that it wasn't consistent with Processes.

This PR removes this page. We will reintroduce it once we're happy with the new approach.  

#### Testing

1. Go to http://localhost:3000/admin/participatory_processes 
2. Click in a process title
3. It should work just like other spaces (i.e. Assemblies), going to the edit page of this space

### :camera: Screenshots

Mockup of the Process show page: 

![Mockup of the Process show page](https://github.com/decidim/decidim/assets/717367/e8166daa-def5-4787-a2c6-88b4a185b9d9)

Current (removed) page: 

![Screenshot of the Process show page](https://github.com/decidim/decidim/assets/717367/24f2864e-f253-457c-8f01-95057342c66b)



:hearts: Thank you!
